device_doctor/lib/src/ios_debug_symbol_doctor.dart

```diff
@@ -50,8 +50,25 @@ class DiagnoseCommand extends Command<bool> {
       return false;
```

Comment:
Maybe rather than returning `false`, we should collect the results of both this AND `DevToolsSecurity` and report both?

Reply:
Sure, I'll just do the `DevToolsSecurity` check first